Resources/Prototypes/_DV/Flavors/flavors.yml

```diff
@@ -116,6 +116,17 @@
   flavorType: Complex
   description: flavor-complex-blellow
 
+- type: flavor
+  id: drgibbbloodred
+  flavorType: Complex
+  description: flavor-complex-drgibbbloodred
+
+- type: flavor
+  id: lemonlimebitters
+  flavorType: Complex
+  description: flavor-complex-lemon-lime-bitters
+
+
```

Comment:
move them back to where it was, this part of the file is for flavors added to upstream drinks